tests/conftest.py

```diff
@@ -176,3 +176,39 @@ def catalog(multi_backend_connection, config) -> AggregatorCollectionCatalog:
         backends=multi_backend_connection,
         config=config
     )
+
+
+@pytest.fixture
+def json_capabilities_no_endpoints():
+    return {
+        "api_version": "1.1.0",
+        "backend_version": "0.6.5a1.dev20221208+820",
+        "description": "OpenEO API to the Terrascope/VITO Remote Sensing product catalog and processing services (using GeoPySpark driver).",
```

Comment:
TODO: avoid references to Terrascope/VITO here, just use dummy names and references

Reply:
Done.